.travis.yml

```diff
@@ -1,10 +1,40 @@
 language: node_js
-node_js:
+node_js: 
   - "8"
   - "10"
-script:
-  - "npm run build"
-  - "npm run lint"
-  - "npm run unit-test"
-  - "node build_release_package_json.js"
-  - "npm run integration-test"
+
+script: npm run unit-test
+
+jobs:
+  include:
+    - stage: build
+      script: npm run build
+    - stage: lint
+      script: npm run lint
+    - stage: Test Integration
+      script:
+        - npm run build
```

Comment:
We unfortunately need to re-run build because travis doesn't keep anything in between stages